spec/models/quote_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+RSpec.describe Quote, type: :model do
+  let(:quote) { create(:quote) }
+
+  it { is_expected.to validate_length_of(:code).is_at_most(255) }
```

Comment:
please split the validations, like:

have_many
have_many
(empty line)
belong_to
belong_to
(empty_line)
.